### DISCUSSION TITLE
Any way to ignore show with no external id?

### DISCUSSION THREAD

**Discussion by aptalca** [arabcoders/watchstate#318] [General] [answered]
I see that there is a command for ignoring a specific external id. But is there a way to ignore a show with no external id?

I have a some videos (P90X workout videos) added to my libraries as a TV show but it has no external id as it isn't cataloged on tvdb or imdb. I manually created the metadata and added it to my emby and plex servers.

When watchstate runs via cron, I get log messages about every single video (a few dozen) on every server, every time. Is there way to tell watchstate to ignore that show perhaps by the internal id?

Thanks

**Comment by arabcoders** [comment, marked as answer]
Hi, sadly it used to be possible before. However, right now it's not possible to do so as i no longer use add the backend id to the GUIDs list. As a workaround you could assign random number for any of the external ids at series level, and then proceed to ignore that.
